pyomo/common/tests/test_unittest.py

```diff
@@ -236,5 +236,198 @@ def test_bound_function_require_fork(self):
+# ==========================================================
+# ----------------------------------------------------------
+#   Problem Information
+# ----------------------------------------------------------
+Problem:
+- Lower bound: -inf
+  Upper bound: inf
+  Number of objectives: 1
+  Number of constraints: 3
+  Number of variables: 3
+  Sense: unknown
+# ----------------------------------------------------------
+#   Solver Information
+# ----------------------------------------------------------
+Solver:
+- Status: ok
+  Message: Ipopt 3.12.3\x3a Optimal Solution Found
+  Termination condition: optimal
+  Id: 0
+  Error rc: 0
+  Time: 0.0408430099487
+# ----------------------------------------------------------
+#   Solution Information
+# ----------------------------------------------------------
+Solution:
+- number of solutions: 1
+  number of solutions displayed: 1
+- Gap: None
+  Status: optimal
+  Message: Ipopt 3.12.3\x3a Optimal Solution Found
+  Objective:
+    f1:
+      Value: -9.99943939749e-05
+  Variable:
+    compl.v:
+      Value: 9.99943939749e-05
+    y:
+      Value: 9.99943939749e-05
+  Constraint: No values
+"""
+
+pass_ref = """
+[    0.00] Setting up Pyomo environment
+[    0.00] Applying Pyomo preprocessing actions
+[    0.00] Creating model
+[    0.00] Applying solver
+[    0.05] Processing results
+    Number of solutions: 1
+    Solution Information
+      Gap: None
+      Status: optimal
+      Function Value: -0.00010001318188373491
+    Solver results file: results.yml
+[    0.05] Applying Pyomo postprocessing actions
+[    0.05] Pyomo Finished
+# ==========================================================
+# = Solver Results                                         =
+# ==========================================================
+# ----------------------------------------------------------
+#   Problem Information
+# ----------------------------------------------------------
+Problem:
+- Lower bound: -inf
+  Upper bound: inf
+  Number of objectives: 1
+  Number of constraints: 3
+  Number of variables: 3
+  Sense: unknown
+# ----------------------------------------------------------
+#   Solver Information
+# ----------------------------------------------------------
+Solver:
+- Status: ok
+  Message: Ipopt 3.14.13\x3a Optimal Solution Found
+  Termination condition: optimal
+  Id: 0
+  Error rc: 0
+  Time: 0.04224729537963867
+# ----------------------------------------------------------
+#   Solution Information
+# ----------------------------------------------------------
+Solution:
+- number of solutions: 1
+  number of solutions displayed: 1
+- Gap: None
+  Status: optimal
+  Message: Ipopt 3.14.13\x3a Optimal Solution Found
+  Objective:
+    f1:
+      Value: -0.00010001318188373491
+  Variable:
+    compl.v:
+      Value: 9.99943939749205e-05
+    x:
+      Value: -9.39395440720558e-09
+    y:
+      Value: 9.99943939749205e-05
+  Constraint: No values
+
+"""
+
+fail_ref = """
```

Comment:
There is only one value different in `pass_ref` and `fail_ref` - do you need the rest of the content around it? I worry that all of the "extra stuff" could cause erroneous failures.

Reply:
The extra stuff is actually intentional: it is testing various parts of the baseline filters.  The `fail_ref` is as big as the `pass_ref` so I could make use of the same baseline.